src/upload.ts

```diff
@@ -47,40 +47,40 @@ export async function upload (args: string[], internal = false) {
   return uploaded
 }
 
-async function uploadEntryToURL ([cid, buffer]: Entry, url: string): Promise<string> {
+async function uploadEntryToURL([cid, buffer]: Entry, url: string): Promise<string> {
   const form = new FormData()
   form.append('hash', cid)
   form.append('data', new Blob([buffer]))
   return await fetch(`${url}/dev-file`, { method: 'POST', body: form })
     .then(handleFetchResult('text'))
     .then(r => {
       if (r !== `/file/${cid}`) {
-        throw new Error(`server returned bad URL: ${r}`)
+        throw new Error(`server returned bad URL: ${String(r)}`)
```

Comment:
Can this be fixed not with `String(r)` coercion, but by typing `r` on line 56? E.g. `.then(r: string`?